.git-blame-ignore-revs

```diff
@@ -0,0 +1,27 @@
+# git blame ignore list.
+#
+# This file contains a list of git hashes of revisions to be ignored by git blame.
+# These revisions are considered "unimportant" in that they are unlikely to be what
+# you are interested in when blaming.
+#
+# Instructions:
+# - Only large (generally automated) reformatting or renaming CLs should be
+#   added to this list. Do not put things here just because you feel they are
+#   trivial or unimportant. If in doubt, do not put it on this list.
+# - Precede each revision with a comment containing the first line of its log.
+#   For bulk work over many commits, place all commits in a block with a single
+#   comment at the top describing the work done in those commits.
+# - Only put full 40-character hashes on this list (not short hashes or any
+#   other revision reference).
+# - Append to the bottom of the file (revisions should be in chronological order
+#   from oldest to newest).
+# - Because you must use a hash, you need to append to this list in a follow-up
+#   CL to the actual reformatting CL that you are trying to ignore.
+#
+# This note was taken from the chromium project file and should scare off
+# anyone that would like to put their manual "refactoring" commit here.
+# This magic should be used sparingly.
+# https://chromium.googlesource.com/chromium/src/+/HEAD/.git-blame-ignore-revs
+
+# Bump Fourmolu formatter to 0.18.0.0
+7c1247f80e2de34bbea52eb07ecb12954ba4d7a5
```

Comment:
This will need to be merged with rebase (actually fast forward) otherwise the hash will not match.

Reply:
I think you can do that manually.  Let me know if you need additional admin privileges.